src/generic/grid.cpp

```diff
@@ -5876,6 +5877,18 @@ void wxGrid::OnKeyDown( wxKeyEvent& event )
                 }
                 break;
 
+            case 'C':
+                if (event.GetModifiers() == wxMOD_CONTROL)
+                {
+                    wxGridCellCoordsArray selections = GetSelectedCells();
```

Comment:
This is problematic because selecting the entire "virtual grid" in the grid sample with, say, a million rows/columns, and pressing `Ctrl-C` will now basically hang the program forever (or maybe crash it with an out of memory error).

We really don't need all the selected cells here and, luckily, since very recently there is an efficient way to get just the first one of them: use `GetSelectedBlocks()` and check if the first selected block is a single cell.

Reply:
Cool changes. But wxGridBlocks::iterator doesn't support pointer expression, should I also add :
```
typedef const value_type* pointer;
pointer operator->() const { return m_it; }
```
to wxGridBlocks::iterator or it was done for some reason?

Reply:
Yes, we could add this, sure, I don't know why did I add only `*` and not `->` to be honest. TIA!